app/screens/home/channel_list/channel_list.tsx

```diff
@@ -111,6 +116,32 @@ const ChannelListScreen = (props: ChannelProps) => {
         return () => back.remove();
     }, [handleBackPress]);
 
+    useEffect(() => {
+        const activeServerUrlObserver = async (servers: ServersModel[]) => {
+            const server = servers?.length ? servers.reduce((a, b) => (b.lastActiveAt > a.lastActiveAt ? b : a)) : undefined;
```

Comment:
Isn't this the same as just `servers.reduce((a, b) => (b.lastActiveAt > a.lastActiveAt ? b : a))` (without the code around it)

Reply:
Corrected